packages/carbon-react/src/components/StructuredList/StructuredList.Skeleton.js

```diff
@@ -0,0 +1,76 @@
+/**
```

Comment:
I think we can remove this and use the exported one from `carbon-components-react` directly (unless did you maybe run into issues with it not being exported? 👀